src/stcal/jump/jump.py

```diff
@@ -340,50 +380,57 @@ def flag_large_events(gdq, jump_flag, sat_flag, min_sat_area=1,
     :return: none
     """
 
-    log.info('Flagging large events (snowballs, showers).')
+    log.info('TEST Flagging large events (snowballs, showers).')
 
     n_showers_grp = []
     n_showers_grp_ellipse = []
     for integration in range(gdq.shape[0]):
         for group in range(1, gdq.shape[1]):
-            if use_ellipses:
-                jump_ellipses = find_ellipses(gdq[integration, group, :, :], jump_flag, min_jump_area)
-                n_showers_grp_ellipse.append(len(jump_ellipses))
-                gdq[integration, group, :, :], num_events = \
-                    extend_ellipses(gdq[integration, group, :, :],
-                                    jump_ellipses, sat_flag, jump_flag,
-                                    expansion=expand_factor)
+            current_gdq = 1.0 * gdq[integration, group, :, :]
+            prev_gdq = 1.0 * gdq[integration, group - 1, :, :]
+            diff_gdq = 1.0 * current_gdq - prev_gdq
```

Comment:
Does the multiplication by "1.0" in these 3 statements turn all of the resulting arrays (current_gdq, prev_gdq, diff_gdq) into floats? If so, why are floats needed for what should be whole integer values?

Reply:
Still need an answer to this, which is the same as Clare's question.